src/main/webapp/search_area.js

```diff
@@ -16,32 +16,52 @@ document.addEventListener("DOMContentLoaded", async function() {
   // This checks that requester User has valid credentials to edit/delete/view ALL organizations. (by checking userID)
   let isMaintainer = false;
   let forOrganizationsPage = false;
-
+  let forEventsPage = false;
   if (document.getElementById('search-area')) {
-    const mainSearchArea = new SearchArea(document.getElementById('search-area'), isMaintainer, forOrganizationsPage);
+    const mainSearchArea = new SearchArea(document.getElementById('search-area'), isMaintainer, forOrganizationsPage, forEventsPage);
     mainSearchArea.handleOrganizations();
   }
-  isMaintainer = true;
   if (document.getElementById('all-organizations')) {
     forOrganizationsPage = true;
-    const organizationSearchArea = new SearchArea(document.getElementById('all-organizations'), isMaintainer, forOrganizationsPage);
+    const organizationSearchArea = new SearchArea(document.getElementById('all-organizations'), isMaintainer, forOrganizationsPage, forEventsPage);
     organizationSearchArea.handleOrganizations();
   }
+  if (document.getElementById('events')) {
+      forEventsPage = true;
+      const eventsSearchArea = new SearchArea(document.getElementById('events'), isMaintainer, forOrganizationsPage, forEventsPage);  }
 });
 
 class SearchArea {
-  constructor(searchAreaElement, isMaintainer, forOrganizationsPage) {
-    this.searchArea = searchAreaElement;
-    this.organizationSearchArea = document.createElement("div");
+  constructor(searchAreaElement, isMaintainer, forOrganizationsPage, forEventsPage) {
+    this.searchAreaContainer = searchAreaElement;
+    this.searchArea = document.createElement("div");
```

Comment:
could you update this on line 47 of filter_tag_area.js?

Reply:
yup its updated